telegram/ext/dispatcher.py

```diff
@@ -652,6 +652,50 @@ def remove_handler(self, handler: Handler, group: int = DEFAULT_GROUP) -> None:
                 del self.handlers[group]
                 self.groups.remove(group)
 
+    def move_chat_data(self, old_id: int, new_id: int, update_values: bool = False, copy_values: bool = False) -> None:
+        """Move chat_data content for old_id to new_id.
+
+        Args:
+            old_id (:obj:`int`, required): The old chat_id
+            new_id (:obj:`int`, required): The new chat_id
+            update_values (:obj:`bool`, optional): if True, dict.update will be used else, an assigment operation will be performed. Set to False by default
+            copy_values (:obj:`bool`, optional): if False, the user_data of old_id will be deleted. Set to True by default
+        """
+        if not isinstance(old_id, int) or not isinstance(new_id, int):
+            raise ValueError("old_id, and new_id must be integers")
+
+        if update_values:
+            self.chat_data.setdefault(new_id, {}).update(self.chat_data.get(old_id, {}))
```

Comment:
`setdefault` and `get` are not necessary as `chat_data` is already a `defaultdict`